pkg/sqlstore/store_user.go

```diff
@@ -13,8 +15,62 @@ type UserColumn struct {
 
 // UserRows defines a row result.
 type UserRows struct {
-	Columns []UserColumn    `json:"columns"`
-	Rows    [][]interface{} `json:"rows"`
+	Columns []UserColumn   `json:"columns"`
+	Rows    [][]*UserValue `json:"rows"`
```

Comment:
I realized that since we're always scanning into `UserValue`, this could be typed now, and that change bubbles up through lots of places giving us strong typing. No need for asserting type in some places now.